src/opnsense/mvc/app/views/OPNsense/Core/reboot.volt

```diff
@@ -0,0 +1,37 @@
+<p><strong>{{ lang._('Are you sure you want to reboot the system?') }}</strong></p>
+
+<button id="do-reboot" class="btn btn-primary">{{ lang._('Yes') }}</button>
+<a href="/" class="btn btn-default">{{ lang._('No') }}</a>
+
+<script>
+    'use strict';
+
+    function rebootWait() {
+        $.ajax({
+            url: '/',
+            timeout: 2500
+        }).fail(function () {
+            setTimeout(rebootWait, 2500);
+        }).done(function () {
+            $(location).attr('href', '/');
+        });
+    }
+
+    $(document).ready(function() {
+        $('#do-reboot').on('click', function() {
+            BootstrapDialog.show({
+                type:BootstrapDialog.TYPE_INFO,
+                title: '{{ lang._('Your device is rebooting') }}',
+                closable: false,
+                onshow: function(dialogRef){
+                    dialogRef.getModalBody().html(
+                        '{{ lang._('The system is rebooting now, please wait…') }}' +
```

Comment:
character change unnecessary. if you have macos you can disable these silly on the fly conversions

```suggestion
                        '{{ lang._('The system is rebooting now, please wait...') }}' +
```

Reply:
This wasn't done on the fly, though, I did it myself. These three dots are an ellipsis, which has its own codepoint. Since we also use proper Unicode for characters like ®, ©, « and », I think using … for ellipses is more consistent (and more correct).

Reply:
you are bouncing a perfectly fine translation

Reply:
Well, I'm not sure if incorrect encoding of a character counts as a fine translation… but I'll change it back.